docs/release-instructions.md

```diff
@@ -1,11 +1,16 @@
 # Steps for creating a new release
 
-1. Create release branch (branch name can be the target version number).
-2. Update version number in `emmopy/__init__.py`.
-3. Commit and push to origin ([emmo-repo/EMMO-python](https://github.com/emmo-repo/EMMO-python.git)).
-4. Push the release branch to GitHub and create a pull request for merging to the master branch.
-5. Once accepted, merge to master.
-6. Create a release on GitHub with a short release description.
-
-   Set the tag to the version number prefixed with `"v"` and title to the version number.
-7. The PyPI distribution package is created automatically upon creation of a new release on GitHub.
+1. Create a release on GitHub with a short release description.
+
+   Ensure you add a `# <version number>` title to the description.
+
+   Set the tag to the version number prefixed with `"v"` and title to the version number as explained above.
+1. Ensure the GitHub Action CD workflows run as expected.
```

Comment:
```suggestion
2. Ensure the GitHub Action CD workflows run as expected.
```

Reply:
Markdown doesn't require one to actually do the numbering. By using `1.` it's easier to "update" the numbering, since it will be dynamic. This was for example a minor hurdle when I was restructuring this list and reusing points with a higher number.

Reply:
But maybe for readability of the raw markdown it's still preferred to actually *do* the numbering?

Reply:
Markdown is smarter than me :) . We can keep it dynamically if you wish, I do not have strong preferences either way.